src/content/docs/infrastructure/host-integrations/installation/update-infrastructure-host-integration-package.mdx

```diff
@@ -69,46 +92,133 @@ Use either of these options to check whether your infrastructure on-host integra
 
```

Comment:
On this section we can add the command to check the package version on a windows server.


On a PowerShell window run the following command to check the version installed:

```
get-package -name "*INTEGRATION_FILE_NAME*" | select name, version
```

mssql integration for example:
```
get-package -name "*nri-mssql*" | select name, version
```